docs/api-reference/supervisor-api.md

```diff
@@ -23,89 +23,3075 @@ sidebar_label: Supervisors
   ~ under the License.
   -->
 
-This document describes the API endpoints to manage and monitor supervisors for Apache Druid.
+This document describes the API endpoints to manage and monitor Supervisors for Apache Druid.
 
-## Supervisors
+In this document, `http://<SERVICE_IP>:<SERVICE_PORT>` is a placeholder for the server address of deployment and the service port. For example, on the quickstart configuration, replace `http://<ROUTER_IP>:<ROUTER_PORT>` with `http://localhost:8888`.
```

Comment:
```suggestion
In this document, `http://<ROUTER_IP>:<ROUTER_PORT>` is a placeholder for the server address of deployment and the service port. For example, on the quickstart configuration, replace `http://<ROUTER_IP>:<ROUTER_PORT>` with `http://localhost:8888`.
```
Does the first placeholder need to say "SERVICE"? Since you use "ROUTER" throughout the doc, I think it is best to keep it consistent.